hack/ci/install-cri-o.sh

```diff
@@ -15,49 +15,22 @@
 
 set -euo pipefail
 
-COMMIT_ID="d7cc66fe80c51a197d13a642ad5bddd9dc7e9f74"
+COMMIT_ID=c4c89c6e426bf574c345465f7fab81b56fcfb1a8
+TAG=v1.23.1
 
-curl "https://raw.githubusercontent.com/cri-o/cri-o/${COMMIT_ID}/scripts/get" | bash
+curl "https://raw.githubusercontent.com/cri-o/cri-o/$COMMIT_ID/scripts/get" | bash -s -- -t "$TAG"
 
 . /etc/os-release
 
 if [[ $ID == fedora ]]; then
-    chcon -u system_u -r object_r -t container_runtime_exec_t \
-        /usr/local/bin/crio \
-        /usr/local/bin/crio-status \
-        /usr/local/bin/runc \
-        /usr/local/bin/crun
-
-    chcon -u system_u -r object_r -t bin_t \
-        /usr/local/bin/conmon \
-        /usr/local/bin/crictl \
-        /usr/local/bin/pinns
-
-    chcon -R -u system_u -r object_r -t bin_t /opt/cni/bin
-
```

Comment:
Why are the `chcon` calls no longer needed?

Reply:
The script already does it :) https://github.com/cri-o/cri-o/pull/5711